docs/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Do you know what caused this modification?

Reply:
https://github.com/mui-org/material-ui-x/pull/3343

Probably this one
I checked the NextJS example and they have this line

Reply:
https://github.com/vercel/next.js/pull/28316/files

I think I found the PR